omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java

```diff
@@ -762,76 +789,39 @@ private void addPaths() {
 			/////////////////////////
 			Path rootPathPostCreate = buildCreatePath(rootPathGetAll, resourceHandler, resourceName,
 			    resourceParentName);
-			addIndividualPath(resourceParentName, resourceName, rootPathPostCreate, "");
+			addIndividualPath(resourceParentName, resourceName, rootPathPostCreate, "", resourceHandler);
 			
 			/////////////////////////
 			// GET with UUID       //
 			/////////////////////////
 			Path uuidPathGetAll = buildGetWithUUIDPath(uuidPath, resourceHandler, resourceName,
 			    resourceParentName);
-			addIndividualPath(resourceParentName, resourceName, uuidPathGetAll, "/{uuid}");
+			addIndividualPath(resourceParentName, resourceName, uuidPathGetAll, "/{uuid}", resourceHandler);
 			
 			/////////////////////////
 			// POST update         //
 			/////////////////////////
 			Path uuidPathPostUpdate = buildUpdatePath(uuidPathGetAll, resourceHandler, resourceName,
 			    resourceParentName);
-			addIndividualPath(resourceParentName, resourceName, uuidPathPostUpdate, "/{uuid}");
+			addIndividualPath(resourceParentName, resourceName, uuidPathPostUpdate, "/{uuid}", resourceHandler);
 			
 			/////////////////////////
 			// DELETE              //
 			/////////////////////////
 			Path uuidPathDelete = buildDeletePath(uuidPathPostUpdate, resourceHandler, resourceName,
 			    resourceParentName);
-			//addIndividualPath(pathMap, uuidPathDelete, resourceParentName, resourceName, uuidPathDelete, "/{uuid}");
 			
 			/////////////////////////
 			// DELETE (purge)      //
 			/////////////////////////
 			Path uuidPathPurge = buildPurgePath(uuidPathDelete, resourceHandler, resourceName,
 			    resourceParentName);
-			addIndividualPath(resourceParentName, resourceName, uuidPathPurge, "/{uuid}");
+			addIndividualPath(resourceParentName, resourceName, uuidPathPurge, "/{uuid}", resourceHandler);
 		}
 	}
 	
-	private void addSubclassOperations() {
```

Comment:
Because it wasn't used to be honest. As a matter of fact that Resources and SubResource Operations are added from the `#addPaths()` method, what stops the Subclass resource operations to be handled there? I just made it possible for the subclass resources to be handled there. So no need of the method

Reply:
And this method was not even referenced anywhere, it needed alot of revamping and more work!